.github/workflows/python-ci-single.yml

```diff
@@ -110,6 +110,19 @@ jobs:
         CC: ${{ inputs.cc }}
         CXX: ${{ inputs.cxx }}
 
+    # Temporary for
+    # https://github.com/single-cell-data/SOMA/pull/222
+    # https://github.com/single-cell-data/TileDB-SOMA/pull/3001
+    - name: Install somacore branch
+      if: ${{contains(github.event.head_commit.message, 'overwrite classes to make queries use C++ re-indexer') }}
+      run: |
+          pip uninstall somacore
+          git clone git@github.com:single-cell-data/SOMA.git soma
+          cd ./SOMA
+          git checkout kerl/python-3.12
+          pip install ./
+          python -c 'import somacore; print(somacore.__version__)'
```

Comment:
`pip install git+https://github.com/single-cell-data/SOMA.git@kerl/python-3.12` might be able to replace this with a one-liner that leaves your venv in a better state?

Reply:
@thetorpedodog thanks! This stanza was my copy-paste of another time someone wanted to run tiledbsoma CI against a non-merged somacore mod ... I'll save off your advice in my cheat-sheet for next time -- thank you!